app/components/Checkbox.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Add basic test files for these two new components. Snapshot tests ok. Functionality Unit tests even better.